python/mathsys/main/builder.py

```
@@ -17,15 +17,18 @@ class Builder:
     # CLASS -> VARIABLES
     filename: str
     data: bytes
+    targets = {
+        "unix/x86/64": "x86_64-unknown-linux-gnu",
+        "web": "wasm32-unknown-unknown"
+    }
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)